src/iframe-helper.js

```diff
@@ -380,3 +380,61 @@ function parseIfNeeded(data) {
   }
   return data;
 }
+
+
+
+/**
+ * Manages a postMessage API for an iframe with a subscription message and
+ * a way to broadcast messages to all subscribed windows, which
+ * in turn must all be descendants of the contentWindow of the iframe.
+ */
+export class SubscriptionApi {
```

Comment:
I think it really depends on if we want expose the requestCallback. If you can foresee in near future we'd have callbacks hooked up other than send(), I think a class here is the right approach.

Otherwise, a send() function is simplifying the code a lot. 
I will leave the decision to you :-).